automation/submission-perf-test.sh

```diff
@@ -9,13 +9,50 @@ WORKING_DIR=${3:-$PWD}
 
 # Check if $ENV is not empty
 if [[ ! ${ENV} ]]; then
-    echo "ENV is empty. Run again \`sh submission-perf-test.sh <alpha or staging>\`"
+    echo
+    "ENV is empty. Run again sh submission-perf-test.sh <alpha or staging"
     exit 1
-else echo "Starting Perf test in $ENV"; fi
-
+else
+    echo "Starting Perf test in {$ENV}"
+fi
 
 JSON_CREDS=`docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN -e VAULT_ADDR=https://clotho.broadinstitute.org:8200 broadinstitute/dsde-toolbox vault read -format=json secret/dsde/firecloud/dev/common/firecloud-account.pem | jq '.data'`
 
+users=(
+     harry.potter@test.firecloud.org
+     ron.weasley@test.firecloud.org
+     mcgonagall.curator@test.firecloud.org
+     draco.malfoy@test.firecloud.org
+     hermione.owner@test.firecloud.org
+     dumbledore.admin@test.firecloud.org
+   )
+
+
+checkToken () {
+    user=$1
+
+    # Verify that user does not need to refresh their token
+    if
+        curl -f -v --silent -X GET --header "Accept: application/json" --header "Authorization: Bearer $ACCESS_TOKEN" "https://firecloud-orchestration.dsde-$ENV.broadinstitute.org/api/refresh-token-status"  2>&1 | grep '"requiresRefresh": true'
+    then
+        echo "This user needs its refresh token refreshed"
+        exit 1
+    fi
+
+}
+
+callbackToNIH() {
+    user=$1
+
+    echo "
+    Launching calback to NIH for:
+    user=$1
+    "
+     ACCESS_TOKEN=`docker run --rm -v $WORKING_DIR:/app/populate -w /app/populate broadinstitute/dsp-toolbox python get_bearer_token.py "${user}" "${JSON_CREDS}"`
+
+      curl -X POST --header "Content-Type: application/json" --header "Accept: application/json" --header "Authorization: Bearer $ACCESS_TOKEN" -d "{\"jwt\":\"eyJhbGciOiJIUzI1NiJ9.ZmlyZWNsb3VkLWRldg.NPXbSpTmAOUvJ1HX85TauAARnlMKfqBsPjumCC7zE7s\"}" "https://firecloud-orchestration.dsde-$ENV.broadinstitute.org/api/nih/callback"
```

Comment:
Might want to check with security folks that it's okay that we're putting a jwt into a public(?) github repo.

Reply:
It is, we had this discussion while we did the same with swatomation, this JWT is only for our internal domains